docs/en/observability/user-experience.asciidoc

```diff
@@ -134,22 +134,42 @@ https://web.dev/vitals/[Core Web Vitals] is a recent initiative from Google to i
 metrics that better categorize good and bad sites by quantifying the real-world user experience.
 This is done by looking at three key metrics: loading performance, visual stability, and interactivity:
 
-[role="screenshot"]
-image::images/web-dev-vitals.png[Web vitals (image source: https://web.dev/vitals)]
-
-Image source: https://web.dev/vitals/[web.dev/vitals]
-
 Largest contentful paint (LCP)::
 Loading performance. LCP is the timestamp when the main content of a page has likely loaded.
 To users, this is the _perceived_ loading speed of your site.
 To provide a good user experience, Google recommends an LCP of fewer than 2.5 seconds.
 footnote:[Source: https://web.dev/lcp/[web.dev]]
 
-First input delay (FID)::
-Load responsiveness. FID measures the time between a user's first interaction with a page, like a click,
-and when the page can respond to those interactions.
-To provide a good user experience, Google recommends a FID of less than 100 milliseconds.
-footnote:[Source: https://web.dev/fid/[web.dev]]
+Interaction to next paint (INP)::
+Responsiveness to user interactions.
+The INP value comes from measuring the latency of all click, tap, and keyboard interactions that happen throughout a single page visit and choosing the longest interaction observed.
+To provide a good user experience, Google recommends an INP of fewer than 200 milliseconds.
+footnote:[Source: https://web.dev/articles/inp[web.dev]]
+
+[NOTE]
+====
+Previous {kib} versions included the metric https://web.dev/fid/[First input delay (FID)] in the User Experience app.
+Starting with version 8.12, FID was replaced with _Interaction to next paint (INP)_.
+The APM Real User Monitoring (RUM) Agent started collecting INP data in version 5.16.0.
+If you use an earlier version of the RUM Agent with {kib} version 8.12 or later, it will _not_ capture INP data
+and there will be _no data_ to show in the User Experience app:
```

Comment:
Should we also call out in this note:

Although FID was replaced by INP, the agent (AFAIK) still collects INP metrics and it can be viewed in custom dashboards and Lens.

Reply:
Ah yes. @vigneshshanmugam confirmed this in https://github.com/elastic/apm-agent-rum-js/pull/1473.

Reply:
Beat him to it by 30 minutes! 😄

Reply:
🏆